src/App.jsx

```diff
@@ -419,6 +463,26 @@ function App() {
               >
                 Sign In
               </AnimatedButton>
+              <Divider/>
+              <AnimatedButton
+                type="submit"
+                onClick={signInWithGoogle}
+                variant="contained"
+                color="primary"
+                sx={buttonStyle}
+              >
+                Sign In With Google
+              </AnimatedButton>
+              <Divider/>
+              <AnimatedButton
+                type="submit"
+                onClick={signInWithFacebook}
+                variant="contained"
+                color="primary"
+                sx={buttonStyle}
+              >
+                Sign In With Facebook
```

Comment:
same comment as of Sign in with google

Reply:
How about we create an issue for both of these?

Reply:
works for me!